### ISSUE TITLE
Change main currency at price check

### ISSUE THREAD

**Issue by maksim789456** [Kvan7/Exiled-Exchange-2#29]
For current moment at bulk trade price check show poe1 main currency (chaos and divine)
![image](https://github.com/user-attachments/assets/641d2a32-7721-433e-b59f-9db525344bd5)
But in poe2 most popular main currency is exalt and alchemy

It turns out that at the moment the buttons are useless because they show none or rare prices

**Comment by Kvan7** [comment]
Wait that is working? I was assuming that wasn't working at all. Awesome that saves me a little bit of time at least

**Comment by Kvan7** [comment]
I haven't touched bulk exchange stuff yet, but for normal price check stuff this shouldn't be a problem since everyone is listing with exalts. Still will edit when I get to this part. I am next working on item mods and parsing so this may still be a while

**Comment by dominikprot** [comment]
This is a bigger problem now - bots/scammers are starting to list everything in divines, because the exchange rate on the official trade website is wrong, making the items seem more cheap than they actually are. If you can not filter for exalts now the search is not usable.

**Comment by jybp** [comment]
Most items directly seem to list from the trade website so you have a good understanding of prices:
![image](https://github.com/user-attachments/assets/106953d1-760f-4fe2-b200-a3f3f64735bf)

But for some like soulcores the UI is different and you can only get chaos/divine listings instead of divs despite having mainly exalt listings on the trade site:
![image](https://github.com/user-attachments/assets/c16a68f4-fb41-4bb8-af91-6b14ba592c8a)

Since price checking is the primary usage IDK if something can be done? Maybe it's some sort of tagging issue? That makes the tool unusable for a lot of items.

**Comment by Kvan7** [comment]
Yeah, that one shown is the bulk exchange. That, while making calls and showing stuff, is doing things with poe ninja and is using poe 1. Note the banner at the top that it hasn't been touched yet

**Comment by Kvan7** [comment]
And note on #128, that is just switching a filter in the results to be "exalted orb" instead of "chaos orb", not fully related to the bulk exchange. I may just remove the bulk calling anything just to reduce confusion

**Comment by jybp** [comment]
> I may just remove the bulk calling anything just to reduce confusion

Sounds reasonable to me. TBH as long as the listing below can be force to exalt it should be fine.
